build.sbt

```diff
@@ -16,25 +16,31 @@ inThisBuild(
         url("https://geirsson.com")
       )
     ),
-    crossScalaVersions := Seq(scala212)
   )
 )
 
 onLoadMessage := s"Welcome to sbt-ci-release ${version.value}"
-
+crossScalaVersions := Nil
```

Comment:
just curious, what does this do that `publish / skip` doesn't already take care of?

Reply:
`+` looks at the `crossScalaVersions`, so we should set `crossScalaVersions` to `Nil`. Otherwise it will end up broadcasting `+` commands, like `publishSigned`.